docs/developer-guide/building-modules.md

```diff
@@ -329,6 +335,13 @@ Just type `gulp`:
 Compiling ./client/scss/main.scss</output>
 </pre>
 
+If the `verify` task fails try to remove your npm_modules and bower_components folder and run obt install again:
```

Comment:
Try to avoid making it personal.

'your npm_modules' -> 'the project's local npm_modules…'